### PR TITLE
[Communication] - Administration - November release changelog

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,37 @@
...
+- Added support for sync and async long running operations
+    - beginCreateReservation
+    - beginPurchaseReservation
+    - beginReleasePhoneNumber
 
-### Breaking Change
+### Breaking Changes
 - Removed credential(CommunicationClientCredential credential) and replaced with 
 accessKey(String accessKey) within CommunicationIdentityClientBuilder
+- `PhoneNumberSearch` renamed to `PhoneNumberReservation`
```
Comment (minnieliu) [inline sdk/communication/azure-communication-administration/CHANGELOG.md:14]: @DominikMe and I added a period to the end of all the points.

**Review by ankitarorabit — CHANGES_REQUESTED** [review]
Do you mind also updating the changelogs for sms/chat/common.. the format is straightforward..

## 1.0.0-beta.3 (2020-11-16)

Updated `@azure/communication-chat` version

**Review by paolamvhz — APPROVED** [review]
I see all the changes on the renames reported here.

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
+## 1.0.0-beta.3 (2020-11-16)
+Updated `azure/communication-chat` version
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: nit: azure-communication-chat (just to keep consistent)

**Review by ankitarorabit — APPROVED** [review]

**Pull request merged**